### PR TITLE
Adding spellcheck, confidence, transcribed notebooks, updated README

### PR DESCRIPTION
**Pull request by schase15**
# Description

Added my notebooks for exploring spell checking, creating the confidence flag, and transcribing the given stories. I also updated the README of the notebooks folder to provide descriptions of each notebook.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by Jwilson1172 — DISMISSED** [review]

File: notebooks/README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 #### Overview of `notebooks` content:
 - `submission_endpoint_interactions`: This Notebook demonstrates the functionality for `submission.py` endpoints and outlines the file structure that is required from the endpoints `UploadFile` type.
-- `squad_score_mvp`: This notebook contains data exploration of training data, generation of MinMaxScaler, and Squad Score formula composition for complexity metric.
\ No newline at end of file
+- `squad_score_mvp`: This notebook contains data exploration of training data, generation of MinMaxScaler, and Squad Score formula composition for complexity metric.
+- `count_spelling_errors`: This notebook contains exploration of various spell check libraries and with the idea that it would help correct transcription errors. For the time being, we did not see enough improvement and consistancy to implement this feature.
```
Comment (lorischl-otter) [inline notebooks/README.md]: Typo: "consistency"

Comment (lorischl-otter) [reply notebooks/README.md]: Can you include a little more on the spellcheck thoughts? Maybe something like: "to explore whether spell check could correct transcription errors, act as a metric for student writing, and/or increase the reliability of other metrics"

**Review by lorischl-otter — CHANGES_REQUESTED** [review]
Caught a small typo and requested a little bit more info in the README. Otherwise looks great. Thanks for getting these uploaded!

File: notebooks/README.md (hunk 1 above)
Comment (schase15) [reply notebooks/README.md]: Fixed. Thank you!

**Review by lorischl-otter — APPROVED** [review]
Thanks for the updates!

**Review by Jwilson1172 — APPROVED** [review]

**Pull request merged**